ui/perfherder/graphs/GraphsContainer.jsx

```diff
@@ -253,8 +264,9 @@ class GraphsContainer extends React.Component {
       tickLabels: { fontSize: 13 },
     };
 
-    const chartPadding = { top: 10, right: 10, bottom: 50 };
+    const chartPadding = { top: 10, bottom: 50 };
     chartPadding.left = this.leftChartPadding;
+    chartPadding.right = this.rightChartPadding;
```

Comment:
You can move `right` and `left` into the `chartPadding` object. There's no need to add those properties after the it's declared :)

Reply:
Ok! Moved them into `chartPadding`.